front/app/containers/Admin/index.tsx

```diff
@@ -174,14 +178,23 @@ const AdminPage = memo<Props & WithRouterProps>(
       >
         <ThemeProvider theme={chartTheme}>
           <Container className={`${className} ${whiteBg ? 'whiteBg' : ''}`}>
-            <Sidebar />
-            <RightColumn
-              className={`${fullWidth && 'fullWidth'} ${
-                noPadding && 'noPadding'
-              }`}
-            >
-              {children}
-            </RightColumn>
+            {!contentBuilderLayoutVisible && (
+              <>
+                <Sidebar />
+                <RightColumn
+                  className={`${fullWidth && 'fullWidth'} ${
+                    noPadding && 'noPadding'
+                  }`}
+                >
+                  {children}
+                </RightColumn>
+              </>
+            )}
+            <Outlet
+              id="app.containers.Admin.contentBuilderLayout"
```

Comment:
I think the general mechanism is fine, though we are already doing something similar where some admin pages are rendered full-width, I think that uses an event stream instead. Could be nice to use a consistent way, but imo not worth to rework it.

But the outlets should be agnostic about their children, so would rather call this `adminFullWidthContent` or something in that direction? Same for the state variable.

Reply:
No time to look it up right now, but I'm pretty sure the idea manager is doing it